dagger/config_finder/config_processor.py

```diff
@@ -48,12 +64,14 @@ def process_pipeline_configs(self):
 
                 _logger.info("Processing task config: %s", task_config_path)
                 task_config = self._load_yaml(task_config_path)
+                task_config, deactivate = self.overwrite_params(task_config)
```

Comment:
What if overwrite params returns None in case of deactivate and then if task_config is None we skip it